local/alpha-sdk.sh

```diff
@@ -49,8 +49,8 @@ Usage:
     --bottlerocket-dir            REQUIRED: The directory of the Bottlerocket checkout that we will
                                   build.
 
-    --variant                     OPTIONAL: The Bottlerocket variant that we will build. Defaults
-                                  to 'aws-dev'.
+    --variants                    OPTIONAL: The space-delimited of Bottlerocket variant that we will
```

Comment:
I think you skipped a word, list or set maybe?

```suggestion
    --variants                    OPTIONAL: The space-delimited list of Bottlerocket variants that we will
```

Reply:
Clarified and fixed.